internal/jobs/launch_instance_aws.go

```diff
@@ -73,9 +71,7 @@ func DoEnsurePubkeyOnAWS(ctx context.Context, args *LaunchInstanceAWSTaskArgs) e
 		return fmt.Errorf("step skipped: %w", err)
 	}
 
-	ctx = ctxval.WithAccountId(ctx, args.AccountID)
-	logger := ctxLogger.With().Int64("reservation_id", args.ReservationID).Logger()
-	logger.Info().Interface("args", args).Msg("Processing pubkey upload AWS job")
+	ctxLogger.Info().Interface("args", args).Msg("Processing pubkey upload AWS job")
```

Comment:
Now that the temporary variable `logger` is gone, could you also rename all `ctxLogger` in all jobs to just `logger`? That is more natural to use, we use this everywhere. The only reason for that was the intermediate variable - Go does not support `variable = &FunctionWhichDoesNotReturnAPointer()` although `to.Ptr` utility function could be used to workaround that.